bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/item/EnrichedItemDTOMapper.java

```diff
@@ -42,12 +42,13 @@
      * @param uri the uri
      * @return item DTO object
      */
-    public static EnrichedItemDTO map(Item item, boolean drillDown, URI uri, Locale locale) {
+    public static EnrichedItemDTO map(Item item, boolean drillDown, String itemTypeFilter, URI uri, Locale locale) {
```

Comment:
We would have to mark this PR "API Breaking", right?
And if we break the API, please let us consider making it more powerful and not just adding a very specific use case (it e.g. already won't cover a situation where you would want to filter for 2 types).

Wdyt about introducing a Predicate instead, which could be used for filtering?

Reply:
The very specific use case was requested by you ;)
No, this change is not breaking the API. As you requested, I used exsisting fields.
Of course we can extend the initial need but my initial intention was just to implement ypour use case.

Can you explain what you mean by Predicate ?

Reply:
> No, this change is not breaking the API.

It is. It is a public method of an exported class. Or am I missing something?

> Can you explain what you mean by Predicate ?

Sure: I refer to a `java.util.function.Predicate`. Have a look at `GroupItem.getMembers(Predicate<Item> filterItem)` as an example.